include/odp/api/spec/traffic_mngr.h

```diff
@@ -1233,6 +1253,12 @@ typedef struct {
 	 * greater levels may be connected to the fan-in of tm_node's with
 	 * numerically smaller levels. */
 	uint8_t level;
+
+	/** The strict priority level assigned to packets going through this
+	 * node. In other words all packets going through node
+	 * will be assigned this priority overriding previous priority.
+	 * in the range 0..max_priority. */
+	uint8_t priority;
```

Comment:
Looking at the text above one thinks that this priority will always set. The fact that the priority gets set only in the "override mode" is hidden in the capability description. The description of this field needs some info on when this value matter and the priority gets overridden.

Reply:
Ack, will improve documentation.